src/Aspire.ProjectTemplates/templates/aspire-starter/9.0/Aspire-StarterApplication.1.Tests/Aspire-StarterApplication.1.Tests.csproj

```diff
@@ -17,11 +17,11 @@
     <PackageReference Include="Aspire.Hosting.Testing" Version="9.0.0" />
     <PackageReference Include="coverlet.collector" Version="6.0.2" Condition=" $(TestFramework) == 'xUnit.net' " />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" Condition=" $(TestFramework) != 'MSTest' " />
-    <PackageReference Include="MSTest" Version="3.4.3" Condition=" $(TestFramework) == 'MSTest' " />
```

Comment:
9.0 will become the unsupported template on next release so we shouldn't update this one.